src/client/test-run/index.js.mustache

```diff
@@ -4,6 +4,10 @@
 
     var origin = location.origin;
 
+    // NOTE: location.origin doesn't exist in IE11 on Windows 10.10240 LTSB
+    if (!origin)
+        origin = location.protocol + '//' + location.host + (location.port ? ':' + location.port : '');
```

Comment:
The `location.host` already contains `port`. `location.host === location.hostname + ':' + location.port`.
I suggest this:
```js
var origin = location.origin || location.protocol + '//' + location.host;
```